README.md

```
@@ -0,0 +1,834 @@
+                </dependencies>
+            </plugin>
+        </plugins>
+    </build>
+</project>
+```
+
+This generates a complete application into the `target/frontend-react` directory.
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.20.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)